### ISSUE TITLE
Look into 3rd party libraries

### ISSUE THREAD

**Issue by kpn678** [kpn678/fitlit#6]

**Comment by jordan-sullivan** [comment]
Hey Team, I looked into both of the third party libraries and...they look awesome and not too terribly difficult to use! I think if we pick one to use it will really help us format our date and time, or make beautiful charts! (I'm leaning towards charts if I have to pick one). I think we could def conquer one together. I found 2 really helpful and clear videos if you haven't  looked yet.  Excited to chat at 1pm tomorrow to hear your thoughts. The line graph example towards the end of the chard vid looked sooo good!
Day.JS: https://www.youtube.com/watch?v=vzGzys3enCE
Chart.JS: https://www.youtube.com/watch?v=sE08f4iuOhA
